ops/delivery-e2e.cloudbuild.yaml

```diff
@@ -0,0 +1,54 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+steps:
+  # Clean up any old Emblem instance cruft.
+  - name: '${_REGION}-docker.pkg.dev/${_PROJECT}/e2e-deployer/e2e-deployer:${_E2E_DEPLOYER_SHA}'
+    entrypoint: /bin/bash
+    args:
+      - ./terraform/cleanup.sh
+    env:
+      - "EMBLEM_BILLING_ACCOUNT=${_EMBLEM_BILLING_ACCOUNT}"
+      - "EMBLEM_ORGANIZATION=${_EMBLEM_ORGANIZATION}"
+      - "PROJECT_NUMBER=$PROJECT_NUMBER"
+
+      # Project IDs
+      - "PROD_PROJECT=${_PROD_PROJECT}"
+      - "STAGE_PROJECT=${_STAGE_PROJECT}"
+      - "OPS_PROJECT=${_OPS_PROJECT}"
+
+  # Deploy a new Emblem instance
+  - name: '${_REGION}-docker.pkg.dev/${_PROJECT}/e2e-deployer/e2e-deployer:${_E2E_DEPLOYER_SHA}'
+    entrypoint: /bin/bash
+    args:
+      - ./setup.sh
+    env:
+      - "EMBLEM_BILLING_ACCOUNT=${_EMBLEM_BILLING_ACCOUNT}"
+      - "EMBLEM_ORGANIZATION=${_EMBLEM_ORGANIZATION}"
```

Comment:
Why is billing account and organization needed?

Reply:
Those env vars are no longer needed; they're only used in the "ephemeral" `clean_project_setup.sh` script.

(I went ahead and removed them.)